### PR TITLE
feat(Quantic): close dropdown when Tab outside and CSS

### PR DESCRIPTION
**Pull request by jelmedini**
Close DropDown when Tab outside.
Remove gray underline text on tabs title in the dropdown.
https://www.loom.com/share/4bf5798524fa44dcaf3cd1c26135570e

### CODE REVIEW

File: packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js [hunk 1]
```diff
@@ -390,4 +390,27 @@ export default class QuanticTabBar extends LightningElement {
...
+   * @returns {Element}
+   */
+  get tabBarDropdownContainer() {
+    return this.template.querySelector('.tab-bar_dropdown');
+  }
+
+  /**
+   * @param {FocusEvent} event
+   */
+
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js:405]:
```suggestion
```

File: packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js [hunk 2]
```diff
@@ -390,4 +390,27 @@ export default class QuanticTabBar extends LightningElement {
...
+   * Returns the tab bar dropdown container element.
+   * @returns {Element}
+   */
+  get tabBarDropdownContainer() {
+    return this.template.querySelector('.tab-bar_dropdown');
+  }
+
+  /**
+   * @param {FocusEvent} event
+   */
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js:404]: Please add a description for this event.

File: packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js [hunk 3]
```diff
@@ -390,4 +390,27 @@ export default class QuanticTabBar extends LightningElement {
...
+
+  onBlur(event) {
+    if (
+      this.isDropdownOpen &&
+      event.relatedTarget instanceof HTMLElement &&
+      this.tabBarDropdownContainer.contains(event.relatedTarget)
+    ) {
+      return;
+    }
+    this.closeDropdown();
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js:414]: I think reverting the condition would make the code easier to read. Perhaps you could also extract the part checking whether the target element is in the dropdown.

```suggestion
    const isTargetElementOutsideDropdown = !(event.relatedTarget instanceof HTMLElement &&
      this.tabBarDropdownContainer.contains(event.relatedTarget));
      
    if (this.isDropdownOpen && isTargetElementOutsideDropdown) {
      this.closeDropdown();
    }
```

**Review by nathanlb — APPROVED** [review]
Nicely done! 👍
BTW It's always nice to have a little screenshot or video when it's a visual change.

**Pull request merged**